source/playbooks/refining-and-improving.rst

```diff
@@ -32,6 +32,37 @@ Use the run timeline to help write an accurate retrospective. Events such as own
 .. image:: ../images/Retro.gif
    :alt: Create and publish retrospective reports.
 
+Metrics
+~~~~~~~
+
+Use metrics to identify key areas where you want to extract valuable insights by measuring performance and improvement. Metrics are enabled when you enable retrospectives. Calibrate the type of metric you want to measure, and once a run is finished you can view the output in the retrospective. You can have multiple metrics configured per playbook and you can edit them at any time. Metrics can be configured based on numeric, time, or value input.
+
+These metrics can be anything that's of interest to you and your team. For example, for a software release playbook you might want to have a metric tracking how many bugs were detected during a run. The output of the metrics you've added is provided in the retrospective. Over time, you can use metrics across retrospectives to examine anomalies and refine goals.
+
+Another example is a support incident playbook. The time to resolution metric can be applied and used to identify areas that need more refinement, such as tasks that might work better if they're split up so goals are reached faster.
+
+When you delete configured metrics from a playbook, the data is not delete but you will no longer see those metrics in the dashboard. In addition, the corresponding metric field will be removed from the retrospective form and from published retrospectives.
+
+Metrics dashboard
+^^^^^^^^^^^^^^^^^
+
+For each run, you can enter a value per metric under the retrospective section. This value can be based on whatever you are measuring, such as number of bugs found per run. 
```

Comment:
@gbochora, I know you've approved this PR (thank you!) - if you'd be open to reviewing this section I'd appreciate it.

Reply:
Looks nice!